examples/go-counter/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/marigold-dev/deku/sdks/deku_go_interop"
+)
+
+type message struct {
```

Comment:
Whaever is our opinon on Go  🤫
I think we should use the go-ish way to manage enumeration in our exemple... only for marketing purpose

```go
type Action int

const (
	Decremenet Action = iota+1
	Increment 
)

type Message struct {
	Action Action
}
```